opentech/apply/funds/templates/funds/includes/batch_update_reviewer_form.html

```diff
@@ -0,0 +1,5 @@
+<div class="modal" id="batch-update-reviewers">
+    <h4>Update Reviewers</h4>
```

Comment:
I know that this matches the original requirements, however, we are really only adding reviewers here, not updating them (no reviewer gets removed by this update). I wonder if we need some more text here to explain what will happen or a different title. Just a thought.